tests/integration/_bootstrap.php

```diff
@@ -10,11 +10,6 @@
 DB::query( "ALTER TABLE $wpdb->posts AUTO_INCREMENT = 5096" );
 
 // Disconnect Promoter to avoid license-related notices.
-
-// Start the posts auto-increment from a high number to make it easier to replace the post IDs in HTML snapshots.
-global $wpdb;
-DB::query( "ALTER TABLE $wpdb->posts AUTO_INCREMENT = 5096" );
```

Comment:
Why remove this?

Reply:
It's not removed. It's just Github file is not refreshed or something.Check the whole file link and it exists.